docker-run-log-pipeline-event.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+PROJECT_NAME="$(<configurations/docker_image_project_name.txt)"
+IMAGE_NAME=$PROJECT_NAME-log-pipeline-event
+
+# Check that the correct number of arguments were provided.
+if [[ $# -ne 4 ]]; then
+    echo "Usage: ./docker-run-log-pipeline-event.sh
```

Comment:
You can use `echo "Usage: $0"` like in other scripts

Reply:
What's the advantage of echo "Usage: $0" i think the latter is more descriptive?

Reply:
$0 expands to the name of the shell script thus helps in case of the script is renamed and the usage statement not updated. Also for consistency with other shell scripts.